lib/web/chartjs/Chart.js

```diff
@@ -0,0 +1,16151 @@
+/*!
```

Comment:
I think we don't need full version of the library. I think the bundled/minified version is enough for us. According to our rules, we can't change 3rd party libraries ( because we will haven't the ability to update them), we use 3rd party libraries as "black box", and operate just with the library by contract. In this case, we don't need full versions

Reply:
Good point, agree with it

Reply:
Done, I removed all files but minified js required to use charts